aiida_vasp/calcs/vasp.py

```diff
@@ -213,9 +213,12 @@ def verify_inputs(self):
             raise InputValidationError(f'Site magnetization requires "LORBIT>=10", value given {_lorbit}')
 
     def _prestore(self):
-        """Set attributes prior to storing."""
+        """
+        Set attributes prior to storing.
+        BONAN: Is this used in any case??
+        """
         super(VaspCalculation, self)._prestore()
-        setattr(self, 'elements', ordered_unique_list(self.inputs.structure.get_ase().get_chemical_symbols()))
+        setattr(self, 'elements', ordered_unique_list(self._structure().get_ase().get_chemical_symbols()))
```

Comment:
Can we avoid using ASE?

Reply:
hmm good point, we can ceratainly replace this with AiiDA's own routing

Reply:
turns out this code is for pre-1.0 era where the attributes of the Calcjob gets stored in the DB which is not the case anymore, since now the class is a `Process`.